### PR TITLE
Added link_views example

### PR DESCRIPTION
**Pull request by charithgunasekara**
### Overview

Add link_views() example

 

### Details
Non

### CODE REVIEW

File: pyvista/plotting/plotting.py [hunk 1]
```diff
@@ -3137,6 +3137,30 @@ def link_views(self, views=0):
             index or if ``views`` is a tuple or a list, link the given
             views cameras.
 
+        Examples
+        --------
+        >>> import numpy as np
+
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3143]:
```suggestion
```

File: pyvista/plotting/plotting.py [hunk 2]
```diff
@@ -3137,6 +3137,30 @@ def link_views(self, views=0):
...
+        >>> import pyvista as pv
+        >>> from pyvista import examples
+
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3146]:
```suggestion
```

File: pyvista/plotting/plotting.py [hunk 3]
```diff
@@ -3137,6 +3137,30 @@ def link_views(self, views=0):
...
+        >>> pv.set_plot_theme("document")
+
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3148]:
```suggestion
```

File: pyvista/plotting/plotting.py [hunk 4]
```diff
@@ -3137,6 +3137,30 @@ def link_views(self, views=0):
...
+        >>> # download mesh
+        >>> mesh = examples.download_cow()
+
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3151]:
```suggestion
```

File: pyvista/plotting/plotting.py [hunk 5]
```diff
@@ -3137,6 +3137,30 @@ def link_views(self, views=0):
...
+        >>> decimated = mesh.decimate_boundary(target_reduction=0.75)
+
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3153]:
```suggestion
```

File: pyvista/plotting/plotting.py [hunk 6]
```diff
@@ -3137,6 +3137,30 @@ def link_views(self, views=0):
...
+        >>> p = pv.Plotter(shape=(1, 2), border=False)
+        >>> p.subplot(0, 0)
+        >>> p.add_text("Original mesh", font_size=24)
+        >>> p.add_mesh(mesh, show_edges=True, color=True)
+        >>> p.subplot(0, 1)
+        >>> p.add_text("Decimated version", font_size=24)
+        >>> p.add_mesh(decimated, color=True, show_edges=True)
+
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3161]:
```suggestion
```

File: pyvista/plotting/plotting.py [hunk 7]
```diff
@@ -3137,6 +3137,30 @@ def link_views(self, views=0):
...
+        >>> p.link_views()  # link all the views
+
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3163]:
```suggestion
```

File: pyvista/plotting/plotting.py [hunk 8]
```diff
@@ -3137,6 +3137,30 @@ def link_views(self, views=0):
...
             views cameras.
 
+        Examples
+        --------
+        >>> import numpy as np
+
+        >>> import pyvista as pv
+        >>> from pyvista import examples
+
+        >>> pv.set_plot_theme("document")
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3147]:
```suggestion
```

File: pyvista/plotting/plotting.py [hunk 9]
```diff
@@ -3137,6 +3137,30 @@ def link_views(self, views=0):
             index or if ``views`` is a tuple or a list, link the given
             views cameras.
 
+        Examples
+        --------
+        >>> import numpy as np
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3142]:
```suggestion
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
Great first contribution. I have some suggestions to fix. And let's discuss to improve this Example more!

**Comment by tkoyama010** [comment]
@charithgunasekara I made an example of how the `link_views` works in the Plotter object.
Could you run and check this code?
If it is fine please use this example.

# Not linked view case.
```python
import pyvista
from pyvista import demos
ocube = demos.orientation_cube()
pl = pyvista.Plotter(shape=(1, 2))
pl.subplot(0, 0)
_ = pl.add_mesh(ocube['cube'], show_edges=True)
_ = pl.add_mesh(ocube['x_p'], color='blue')
_ = pl.add_mesh(ocube['x_n'], color='blue')
_ = pl.add_mesh(ocube['y_p'], color='green')
_ = pl.add_mesh(ocube['y_n'], color='green')
_ = pl.add_mesh(ocube['z_p'], color='red')
_ = pl.add_mesh(ocube['z_n'], color='red')
pl.camera_position = 'yz'
pl.subplot(0, 1)
_ = pl.add_mesh(ocube['cube'], show_edges=True)
_ = pl.add_mesh(ocube['x_p'], color='blue')
_ = pl.add_mesh(ocube['x_n'], color='blue')
_ = pl.add_mesh(ocube['y_p'], color='green')
_ = pl.add_mesh(ocube['y_n'], color='green')
_ = pl.add_mesh(ocube['z_p'], color='red')
_ = pl.add_mesh(ocube['z_n'], color='red')
pl.show_axes()
pl.show()
```
a

# Linked view case.
```python
pl = pyvista.Plotter(shape=(1, 2))
pl.subplot(0, 0)
_ = pl.add_mesh(ocube['cube'], show_edges=True)
_ = pl.add_mesh(ocube['x_p'], color='blue')
_ = pl.add_mesh(ocube['x_n'], color='blue')
_ = pl.add_mesh(ocube['y_p'], color='green')
_ = pl.add_mesh(ocube['y_n'], color='green')
_ = pl.add_mesh(ocube['z_p'], color='red')
_ = pl.add_mesh(ocube['z_n'], color='red')
pl.camera_position = 'yz'
pl.subplot(0, 1)
_ = pl.add_mesh(ocube['cube'], show_edges=True)
_ = pl.add_mesh(ocube['x_p'], color='blue')
_ = pl.add_mesh(ocube['x_n'], color='blue')
_ = pl.add_mesh(ocube['y_p'], color='green')
_ = pl.add_mesh(ocube['y_n'], color='green')
_ = pl.add_mesh(ocube['z_p'], color='red')
_ = pl.add_mesh(ocube['z_n'], color='red')
pl.show_axes()
pl.link_views()
pl.show()
```
b

File: pyvista/plotting/plotting.py [hunk 10]
```diff
@@ -3136,7 +3136,55 @@ def link_views(self, views=0):
             If ``views`` is int, link the views to the given view
             index or if ``views`` is a tuple or a list, link the given
             views cameras.
+        Examples
+        ---------
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3140]: Adding one blank line.
```suggestion

        Examples
        ---------
```

File: pyvista/plotting/plotting.py [hunk 11]
```diff
@@ -3136,7 +3136,55 @@ def link_views(self, views=0):
...
+        >>> _ = pl.add_mesh(ocube['x_n'], color='blue')
+        >>> _ = pl.add_mesh(ocube['y_p'], color='green')
+        >>> _ = pl.add_mesh(ocube['y_n'], color='green')
+        >>> _ = pl.add_mesh(ocube['z_p'], color='red')
+        >>> _ = pl.add_mesh(ocube['z_n'], color='red')
+        >>> pl.show_axes()
+        >>> pl.show()
 
+        2) Linked view case.
+        >>> pl = pyvista.Plotter(shape=(1, 2))
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3167]:
```suggestion

        >>> pl = pyvista.Plotter(shape=(1, 2))
```

File: pyvista/plotting/plotting.py [hunk 12]
```diff
@@ -3136,7 +3136,55 @@ def link_views(self, views=0):
             If ``views`` is int, link the views to the given view
             index or if ``views`` is a tuple or a list, link the given
             views cameras.
+        Examples
+        ---------
+        1) Not linked view case.
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3141]:
```suggestion
        Not linked view case.

```

File: pyvista/plotting/plotting.py [hunk 13]
```diff
@@ -3136,7 +3136,55 @@ def link_views(self, views=0):
...
+        >>> _ = pl.add_mesh(ocube['x_p'], color='blue')
+        >>> _ = pl.add_mesh(ocube['x_n'], color='blue')
+        >>> _ = pl.add_mesh(ocube['y_p'], color='green')
+        >>> _ = pl.add_mesh(ocube['y_n'], color='green')
+        >>> _ = pl.add_mesh(ocube['z_p'], color='red')
+        >>> _ = pl.add_mesh(ocube['z_n'], color='red')
+        >>> pl.show_axes()
+        >>> pl.show()
 
+        2) Linked view case.
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3166]:
```suggestion
        2) Linked view case.

```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
@charithgunasekara I have some suggestions but it is small. Thanks a lot.

File: pyvista/plotting/plotting.py [hunk 14]
```diff
@@ -3137,6 +3137,58 @@ def link_views(self, views=0):
...
+        >>> _ = pl.add_mesh(ocube['x_p'], color='blue')
+        >>> _ = pl.add_mesh(ocube['x_n'], color='blue')
+        >>> _ = pl.add_mesh(ocube['y_p'], color='green')
+        >>> _ = pl.add_mesh(ocube['y_n'], color='green')
+        >>> _ = pl.add_mesh(ocube['z_p'], color='red')
+        >>> _ = pl.add_mesh(ocube['z_n'], color='red')
+        >>> pl.show_axes()
+        >>> pl.show()
+
+        2) Linked view case.
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:3168]:
```suggestion
        Linked view case.
```

**Review by akaszynski — APPROVED** [review]
LGTM. Thanks @charithgunasekara and @tkoyama010!

**Pull request merged**